web/models/role_skill.ex

```diff
@@ -15,8 +17,9 @@ defmodule CodeCorps.RoleSkill do
   """
   def changeset(struct, params \\ %{}) do
     struct
-    |> cast(params, [:role_id, :skill_id])
+    |> cast(params, [:cat, :role_id, :skill_id])
     |> validate_required([:role_id, :skill_id])
+    |> validate_inclusion(:cat, cats)
```

Comment:
I would put the `cast` and validation into an `import_changeset` instead. We don't do imports yet, but that would be the ideal place for this.

I added the same migration in my not-yet-merged branch, so I can remove that when this is merged.